### PR TITLE
Add project_license

### PR DESCRIPTION
**Pull request by rrigby**
From the Xmoto source code:
XMOTO is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

Categories added from spec:
https://standards.freedesktop.org/menu-spec/latest/apas02.html

### CODE REVIEW

File: org.tuxfamily.XMoto.appdata.xml [hunk 1]
```diff
@@ -27,6 +28,10 @@ SentUpstream: 2014-09-25
     </p>
   </description>
   <url type="homepage">http://xmoto.sourceforge.net/</url>
+  <categories>
```
Comment (TingPing) [inline org.tuxfamily.XMoto.appdata.xml]: Again don't need these, they go in the `.desktop` file.

Comment (rrigby) [reply org.tuxfamily.XMoto.appdata.xml]: AppStream and desktop files serve different purpose though. There's a valid usecase for both and majority of the applications currently on flathub have category data in their appstream data.

Comment (TingPing) [reply org.tuxfamily.XMoto.appdata.xml]: Appstream extracts info from desktop files at build time.

Comment (rrigby) [reply org.tuxfamily.XMoto.appdata.xml]: In that case then I will remove the categories.

**Comment by TingPing** [comment]
@rrigby Thanks for catching all of these missing licenses!

**Pull request merged**